app/views/merchants/items/index.html.erb

```diff
@@ -1,9 +1,23 @@
 <h2><%= @merchant.name %></h2>
 
-<h3>My Items</h3>
+<section id="enabled">
+  <h3>Enabled Items</h3>
 
-<% @items.each do |item| %>
-  <div id="item-<%= item.id %>">
-    <p><a href="/merchants/<%= @merchant.id %>/items/<%= item.id %>" ><%= item.name %></a></p>
-  </div>
-<% end %>
\ No newline at end of file
+  <% @items.enabled_only.each do |item| %>
+    <div id="item-<%= item.id %>">
+      <p><a href="/merchants/<%= @merchant.id %>/items/<%= item.id %>" ><%= item.name %></a></p>
```

Comment:
Is this line creating a link? Should the syntax be link_to instead of "a href"?

Reply:
Yes! Thanks for the catch. I will update.

Reply:
Updated!